### PR TITLE
Fixes #21086: Create a package for people having an external DB

### PR DESCRIPTION
**Pull request by peckpeck**
https://issues.rudder.io/issues/21086

### CODE REVIEW

File: rudder-external-db/Makefile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.DEFAULT_GOAL := build
+# Used on debian to redirect calls to final Makefile so dh auto commands work
+%:
+	make -C BUILD/rudder/relay/sources/ --debug $@
```
Comment (amousset) [inline rudder-external-db/Makefile:4]: probably not the right command

**Comment by peckpeck** [comment]
PR updated with a new commit

File: rudder-external-db/SOURCES/external-db.conf [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Please fill the configuration to make sure Rudder serveur will be properly installed
```
Comment (amousset) [inline rudder-external-db/SOURCES/external-db.conf:1]:
```suggestion
# Please fill the configuration to make sure Rudder server will be properly installed
```

File: rudder-external-db/SOURCES/external-db.conf [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+# Please ask your database administrator for those informations.
```
Comment (amousset) [inline rudder-external-db/SOURCES/external-db.conf:2]:
```suggestion
# Please ask your database administrator for this information.
```

File: rudder-server/SPECS/rudder-server.spec [hunk 4]
```diff
@@ -79,7 +79,7 @@ Obsoletes: rudder-reports < 7.2
 
 Requires: %(../format-dependencies rpm %{old_epoch}:%{real_version} rudder-agent)
 Requires: %(../format-dependencies rpm %{real_version} rudder-relay)
-Requires: postgresql-server >= 10.3, %{apache}, %{apache_tools}, git-core, iproute, rsync, openssl, %{ldap_clients}, curl, acl, rudder-api-client
+Requires: (postgresql-server >= 10.3 or rudder-external-db), %{apache}, %{apache_tools}, git-core, iproute, rsync, openssl, %{ldap_clients}, curl, acl, rudder-api-client
```
Comment (amousset) [inline rudder-server/SPECS/rudder-server.spec:82]: is it supported on amazon linux 2?

Comment (peckpeck) [reply rudder-server/SPECS/rudder-server.spec:82]: no

**Comment by peckpeck** [comment]
PR updated with a new commit

**Review by amousset — APPROVED** [review]

**Pull request merged**